### PR TITLE
Add missing allow_if_all_abstain parameter

### PR DESCRIPTION
**Pull request by Nightbr**
Missing documentation on access_decision_manager

### CODE REVIEW

File: security/voters.rst [hunk 1]
```diff
@@ -316,5 +317,6 @@ security configuration:
         $container->loadFromExtension('security', array(
             'access_decision_manager' => array(
                 'strategy' => 'unanimous',
+                'allow_if_all_abstain' => true
```
Comment (jmsche) [inline security/voters.rst]: I would add a comma at the end of the line

**Review by jmsche — CHANGES_REQUESTED** [review]

**Review by javiereguiluz — APPROVED** [review]
Even if setting `allow_if_all_abstain` to `true` won't be the best decision in all cases, the name of the option is self-explanatory and readers won't have a problem adapting it to their needs.

**Comment by javiereguiluz** [comment]
@Nightbr thanks for this! I've merged it in the 2.7 branch instead, so GitHub may make it look closed instead of merged ... but it's merged.

**Comment by xabbuh** [comment]
I think we should have set it to `false` here. Above the config examples we always talk about permission is only granted if *all*  voters grant access. However, setting `allow_if_all_abstain` to `true` will have the opposite effect if no voter makes (is able to make) a decision.

**Pull request closed without merging**